README.md

```diff
@@ -48,10 +105,8 @@ Dynamic Extensions is systematically integration tested against:
 * Alfresco Enterprise 5.1
 * Alfresco Enterprise 5.0
 
-### Known Alfresco issues that impact Dynamic Extensions
-
-#### Alfresco 6.1 - wrong version of 'Commons annotations' used
-When using DE on Alfresco 6.1, it is possible that it fails to startup due to following error:
+#### Known Alfresco issues that impact Dynamic Extensions</summary>
+<details><summary>Alfresco 6.1 - wrong version of 'Commons annotations' used</summary>When using DE on Alfresco 6.1, it is possible that it fails to startup due to following error:

```

Comment:
You can use a header in the `<summary>` I think

Reply:
I tried locally, didn't work...
(Which could also be an issue with my local markdown viewer / formatter)